projects/angular/components/ui-grid/src/ui-grid.component.html

```diff
@@ -253,16 +254,9 @@
         </div>
     </div>
     <div class="ui-grid-footer-container">
-        <mat-paginator *ngIf="!!footer"
-                       [pageIndex]="footer?.state?.pageIndex"
-                       [pageSize]="footer?.state?.pageSize"
-                       [pageSizeOptions]="footer?.pageSizes"
-                       [length]="footer?.length"
-                       [disabled]="disabled"
-                       [showFirstLastButtons]="true"
-                       [hidePageSize]="footer?.hidePageSize"
-                       (page)="footer?.pageChange?.next($event)">
-        </mat-paginator>
+        <ng-container *ngIf="dataManager?.useAlternateDesign; else classicFooter">
```

Comment:
so, this would become something like:
```suggestion
        <ng-container *ngIf="useAlternateDesign; else classicFooter">
```